src/renderers/shared/stack/event/ReactSyntheticEventType.js

```diff
@@ -8,14 +8,29 @@
  *
  * Flow type for SyntheticEvent class that includes private properties
  *
- * @providesModule ReactSyntheticEvent
+ * @providesModule ReactSyntheticEventType
  * @flow
  */
 
 'use strict';
 
-export type DispatchConfig = any;
+import type {ReactInstance} from 'ReactInstanceType';
+
+export type DispatchConfig = {
+  dependencies: Array<string>,
+  phasedRegistrationNames?: {
+    bubbled: string,
+    captured: string,
+  },
+  registrationName?: string,
+};
 
 export class ReactSyntheticEvent extends SyntheticEvent {
   dispatchConfig: DispatchConfig;
+  getPooled: (
```

Comment:
I think that adding `@flow` to `SyntheticEvent` would kill two birds in one stone :)